lib/class-wp-theme-json-gutenberg.php

```diff
@@ -404,6 +405,7 @@ class WP_Theme_JSON_Gutenberg {
 			'textDecoration' => null,
 			'textTransform'  => null,
 		),
+		'behaviorsUIEnabled'            => null,
```

Comment:
Same as [below](https://github.com/WordPress/gutenberg/pull/49972/files#r1201361853).

My instinct is to be explicit and modular here. Do we assume that theme builders will understand any implied semantics in `-UIEnabled`?

Rather, an independent unit `behavior.lightbox` tells us what we're dealing with immediately. 

If the term "behavior" doesn't have the semantic punch to convey what the sub properties are doing, then it's okay to stop and spend time thinking about it. It's far better than doing the backwards compatibility cha-cha down the road. 😄